src/integrations/contact-input.tsx

```diff
@@ -293,7 +298,7 @@ const ContactInput: FC<ContactInput> = ({
 						customComponent: <Loader />
 					}
 				]);
-				new Promise((resolve, reject) => {
+				new Promise((resolve, _reject) => {
```

Comment:
why this renaming?

Reply:
because in line 29 reject is imported from lodash

Reply:
ok, I don't like the naming :(  I would prefer a renaming of lodash import or a renaming of this callback (something like `promiseReject`

Reply:
done